### DISCUSSION TITLE
Terms of service: Question about github pages for commercial use

### DISCUSSION THREAD

**Discussion by a-fragos** [community/community#28446] [New to GitHub]
Hello everyone,

In github's term of service for github pages one can find:

> GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either facilitating commercial transactions or providing commercial software as a service (SaaS). GitHub Pages sites shouldn't be used for sensitive transactions like sending passwords or credit card numbers.

My question is - very generally - the following: If i make a commercial site, in the sense that it advertises a product, but it doesn' t facilitate any online payments / selling or account logging, would that be violating the terms and conditions of github?

To be more clear, if i make a site that advertises a product and the payment is done either in another page (not github related) by redirecting, or by the customer (after communicating with me, by sending an email in an address i will have in my page), would that be violating the terms and conditions of github?

Thank you in advance for replying. Sorry to bother you, but i didn't find an answer that answers specifically this kind of question.
AFragos.

**Comment by radar-blip** [comment]
They would get you on the 'redirecting'. And probably want their fair share lol.

**Reply by a-fragos** [reply]
If redirecting is the only problem, i can exclude this method of contacting / paying.

**Reply by radar-blip** [reply]
Sounds like you want to exclude the middleman, and its a pretty big middleman. You could try I guess. 👍🏼

**Reply by radar-blip** [reply]
My status thingy will give you the short answer.

**Comment by AmmarHaddadi** [comment]
I kinda have the same problem
I have a company that provides a service , on the website I want to list our info, experiences, career,  etc ... but also **pricing** for our services , however there are no transactions or any bank related thing, literally just the prices, numbers
got no idea if it is against the rules
